src/main/java/com/bravo/user/dao/specification/AbstractSpecification.java

```diff
@@ -68,6 +68,17 @@ protected void applyDateTimeFilter(
     predicates.add(path.in(values));
   }
 
+  protected <X, Y  extends Collection<X>> void applyInLengthFilter(
```

Comment:
1. This `AbstractSpecification` is only used for searching entities so validation at this layer is not practical. 
2. Although nice use of Generics +1.